### ISSUE TITLE
`wp core verify-checksums` without installation

### ISSUE THREAD

**Issue by szepeviktor** [wp-cli/wp-cli#2086]
I am hunting a malware on shared hosting. I've downloaded all files but no db.

How would it be possible to run `wp core verify-checksums --locale=.. --version=...` without db?

**Comment by danielbachhuber** [comment]
We'd have to do away with `wp_http_supports()` in `get_core_checksums()`, which should be easy enough to do because `Utils\http_request()` will warn if a https request failed, but try again.

However, we can't easily run on `@when before_wp_load` because we're dependent on `global $wp_version, $wp_local_package`. The former could be sniffed out of the version file, but the latter is populated by the database.

Ideas?

**Comment by szepeviktor** [comment]
I was thinking about `@when before_wp_load` also.

**Comment by szepeviktor** [comment]
Is it possible to run before **and** after WP?

**Comment by danielbachhuber** [comment]
> Is it possible to run before and after WP?

Not easily. We'd need #2089 I think.

**Comment by szepeviktor** [comment]
Thank you very much.

**Comment by szepeviktor** [comment]
For googlers. To verify checksums without installation:
```
wp core verify-checksums --version=4.3 --locale=en_US
```

**Comment by szepeviktor** [comment]
@danielbachhuber 
```
$ wp --allow-root --info
PHP binary:     /usr/bin/php5
PHP version:    5.6.13-0+deb8u1
php.ini used:   /etc/php5/cli/php.ini
WP-CLI root dir:        phar://wp-cli.phar
WP-CLI global config:   /root/.wp-cli/config.yml
WP-CLI project config:
WP-CLI version: 0.20.2

$ wp --allow-root core verify-checksums --version=4.3 --locale=en_US
Error: wp-config.php not found.
Either create one manually or use `wp core config`.
```
WHat am I doming wrong?

**Comment by szepeviktor** [comment]
Could it be that this feature is not released?

**Comment by danielbachhuber** [comment]
It isn't released yet. It will be included in v0.21.0:

![image](https://cloud.githubusercontent.com/assets/36432/10561594/f74846de-74e4-11e5-9063-14ff818e6642.png)

You can install nightly with `wp cli update --nightly`
